### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by testy-mbhartley**
testing PR commenting

### CODE REVIEW

**Comment by mbhartleytest** [comment]
PR comment abuse

**Comment by mbhartleytest** [comment]
PR comment disruptive and off topic

**Comment by mbhartleytest** [comment]
PR comment lock/close

**Comment by testy-mbhartley** [comment]
closing now now now locking

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# test-comment-minimize
\ No newline at end of file
+# test-comment-minimize
+
+update for PR testing
```
Comment (mbhartley) [inline README.md:3]: spammy commit comment

**Pull request closed without merging**